src/components/PeopleFilters.tsx

```diff
@@ -1,96 +1,120 @@
-            >
-              17
-            </a>
+  return (
+    <div className="column is-7-tablet is-narrow-desktop">
+      <nav className="panel">
+        <p className="panel-heading">Filters</p>
 
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
+        <p className="panel-tabs" data-cy="SexFilter">
+          {Object.entries(SexFilter).map(([key, value]) => (
+            <Link
+              key={value}
+              className={classNames({
+                'is-active': sex === value || (!sex && value === 'all'),
+              })}
+              to={{
+                search: getSearchWith(searchParams, {
+                  sex: value !== 'all' ? value : null,
+                }),
+              }}
             >
-              18
-            </a>
+              {key}
+            </Link>
+          ))}
+        </p>
 
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
+        <div className="panel-block">
+          <p className="control has-icons-left">
+            <input
+              data-cy="NameFilter"
+              type="search"
+              className="input"
+              placeholder="Search"
+              value={query}
+              onChange={queryHandler}
+            />
 
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
-          </div>
+            <span className="icon is-left">
+              <i className="fas fa-search" aria-hidden="true" />
+            </span>
+          </p>
+        </div>
 
-          <div className="level-right ml-4">
-            <a
-              data-cy="centuryALL"
-              className="button is-success is-outlined"
-              href="#/people"
-            >
-              All
-            </a>
+        <div className="panel-block">
+          <div
+            className="level is-flex-grow-1 is-mobile"
+            data-cy="CenturyFilter"
+          >
+            <div className="level-left">
+              {Object.values(CenturiesFilter).map(century => (
+                <Link
+                  key={century}
+                  data-cy="century"
+                  className={classNames('button mr-1', {
+                    'is-info': centuries.includes(century),
+                  })}
+                  to={{
+                    search: getSearchWith(searchParams, {
+                      centuries: centuries.includes(century)
+                        ? centuries.filter(c => c !== century)
+                        : [...centuries, century],
+                    }),
+                  }}
+                >
+                  {century}
+                </Link>
+              ))}
```

Comment:
Similar to the previous comment, this part of the code where you're mapping over CenturiesFilter can be extracted into a separate component. This would improve readability and make your code more modular.